packages/datagovmy-ui/src/components/Card/PublicationCard.tsx

```diff
@@ -17,18 +17,30 @@ export type Publication = {
 interface PublicationCardProps {
   onClick: () => void;
   publication: Publication;
+  sendAnalytics: (id: string, type: string, event: string) => void;
 }
 
-const PublicationCard: FunctionComponent<PublicationCardProps> = ({ onClick, publication }) => {
+const PublicationCard: FunctionComponent<PublicationCardProps> = ({
+  onClick,
+  publication,
+  sendAnalytics,
+}) => {
   const { t, i18n } = useTranslation(["publications", "common"]);
   const diffInDays = DateTime.now().diff(DateTime.fromISO(publication.release_date), ["days"]);
 
+  const handleClick = () => {
+    onClick();
+    if (typeof sendAnalytics === "function") {
```

Comment:
wondering why would you need to check `typeof sendAnalytics` here ?

Reply:
sorry for the confusion, the check for typeof sendAnalytics is unnecessary in this context, my bad for not removing it before commit